src/components/GptMovieSuggesstions.tsx

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from "react-redux";
+import { RootState } from "../utils/appStore";
+import MovieList from "./MovieList";
+
+const GptMovieSuggesstions: React.FC = () => {
+  const { movies, gptMovies } = useSelector((store: RootState) => store.gpt);
+  if (!gptMovies) return null;
+
+  return (
+    <div className="p-4 mt-2 mx-3 bg-black text-white bg-opacity-70">
+      <div>
```

Comment:
why use an extra div? There are a lot of unnecessary nested divs

Reply:
resolved in this component